public/content/desci/index.md

```diff
@@ -105,10 +105,10 @@ Explore projects and join the DeSci community.
 - [Bio.xyz: get funded for your biotech DAO or desci project](https://www.bio.xyz/)
 - [Fleming Protocol: open-source data economy that fuels collaborative biomedical discovery](http://flemingprotocol.io/)
 - [Active Inference Institute](https://www.activeinference.org/)
-- [CureDAO: Community-Owned Precision Health Platform](https://docs.curedao.org/)
 - [IdeaMarkets: enabling decentralized scientific credibility](https://ideamarket.io/)
 - [DeSci Labs](https://www.desci.com/)
 - [ValleyDAO: an open, global community offering funding and translational support for synthetic biology research](https://www.valleydao.bio)
+- [Cerebrum DAO: sourcing and nurturing solutions to advance brain health and prevent neurodegeneration](https://www.cerebrumdao.com/)
```

Comment:
Link doesn't work @AMelhede 

<img width="1081" alt="Screenshot 2024-05-30 at 15 06 55" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/94f7044a-11ea-4ca5-bb70-9587d4572534">

Reply:
Hi @minimalsm, thanks! This is the first time I ever saw this. I just checked and, yes, still down. It is the official website for Cerebrum DAO, so it is strange to see. Will update as soon as I see it up again

Reply:
It should be working now. Can you recheck?